holidays/financial/ny_stock_exchange.py

```
@@ -285,3 +655,126 @@ class NewYorkStockExchangeStaticHolidays:
+        1992: (
+            (NOV, 27, close_2pm_label % "Day after Thanksgiving Day"),
+            (DEC, 24, close_2pm_label % name_christmas_eve),
+        ),
+        1994: (FEB, 11, close_2_30pm_label % name_snowstorm),
+        1996: (JUL, 5, close_1pm_label % name_after_independence),
+        1997: (DEC, 26, close_1pm_label % name_friday_after_christmas),
+        1999: (DEC, 31, close_1pm_label % "New Year's Eve"),
+        2002: (JUL, 5, close_1pm_label % name_after_independence),
+        2003: (DEC, 26, close_1pm_label % name_friday_after_christmas),
+        2013: (JUL, 3, close_1pm_label % "Day before Independence Day"),
+    }
```

Ruff (0.14.7)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)